src/frontend/src/utils/webAuthnErrorUtils.ts

```diff
@@ -0,0 +1,8 @@
+/** Checks whether the error corresponds with the WebAuthnSpec for duplicate device:
+ *  * https://www.w3.org/TR/webauthn-2/#sctn-createCredential (Step 20)
+ *  * https://www.w3.org/TR/webauthn-2/#sctn-op-make-cred (Step 3)
```

Comment:
Looking at these, there are no guarantees that this is actually a "duplicate" error, right?

Reply:
No, unfortunately not. However, this is also something that [other libraries](https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/browser/src/helpers/identifyRegistrationError.ts#L45) do (because it's the next best thing).